contracts/cw1155-base/src/contract_tests.rs

```diff
@@ -0,0 +1,680 @@
+// #![cfg(test)]
```

Comment:
I haven't seen any other contract tests - so would expect these to be implemented and fixed before merge to be sure of the contract behaviour.

Reply:
Hi Ethan, sorry this is still super WIP. I wanted to get feedback on the state model before I add all the tests so that if we end up changing the state file, not too much code needs to be changed.

Reply:
ha, ethan is @ethanfrey - first time i've ever had a name collision with this handle

Reply:
I am so sorry, I saw frey and assumed incorrectly. Sorry about that.

Reply:
🙃 no worries haha